great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -215,3 +259,82 @@ def _convert_checks_into_output_message(
         output_message += "\n"
 
         return output_message
+
+    @staticmethod
+    def _check_input_validation(
+        expectation_instance,
+        examples: List[ExpectationTestDataCases],
+    ) -> ExpectationDiagnosticCheckMessage:
+        """Check that the validate_configuration method returns True"""
+        passed = False
+        try:
+            first_test = examples[0]["tests"][0]
+        except IndexError:
+            # No examples, so can't get kwargs for ExpectationConfiguration
+            pass
+        else:
+            expectation_config = ExpectationConfiguration(
+                expectation_type=expectation_instance.expectation_type,
+                kwargs=first_test.input,
+            )
+            try:
+                passed = expectation_instance.validate_configuration(expectation_config)
+            except:
```

Comment:
Can we narrow this `except` clause (and all the other ones)? I'm wary of generic catches like this if they can be avoided.

Additionally, I think logging also has applicability here so we know where `validate_configuration` went wrong.